packages/Table/src/Table.js

```diff
@@ -21,8 +25,23 @@ export default function Table(props) {
     ColumnDefinitions = [extractedColumnDefinitions];
   }
 
+  const qty = {
+    columnsLength: extractedColumnDefinitions.length,
+    rowsLength: data.length,
+  };
+
+  let arrowKeyNavigationProps = {};
+  if (enableArrowKeyNavigation) {
+    arrowKeyNavigationProps = {
+      onFocus: handleFocus({ refFocus, tableId }),
+      onBlur: handleBlur({ refFocus, tableId }),
+      onKeyDown: handleKeyDown({ refFocus, tableId, ...qty }),
+      onClick: handleClick({ refFocus, tableId }),
+    };
+  }
+
   return (
-    <sc.Table aria-label={a11yText} {...moreProps}>
+    <sc.Table aria-label={a11yText} id={tableId} tabIndex={0} {...moreProps} {...arrowKeyNavigationProps}>
```

Comment:
Should the tabIndex value be set on the `td` element dynamically to let the screen reader read out the cell content? or maybe there's a better way to support screen reader?

Reply:
yes!!! I saw that approach in the w3c, but let me find out how will be work within the inline editing, because we might have edition on popover, takeover, side panels, so don't really know yet who should control the tabindex yet or what should hold the focus. the editing doesn't happened directly on the td.

Reply:
you know let me change it to the `td` I can go back and change the official way of doing it.

Reply:
Thank you! It would be super nice, hopefully not too much refactoring, I guess later on if there's input in a popover we can just move the focus to that input, arrow key doesn't work in that case, but it's totally fine I think.  It's a different workflow

Reply:
ok fixed it!

Reply:
was quite simple to change it actually good think you catch it.